public/sketch.js

```diff
@@ -124,35 +118,31 @@ function draw() {
     let user = users[id];
     let username = user.username;
 
-    fill(user.color.r, user.color.g, user.color.b);
+    fill(user.color);
 
     // if you haven't ended
-    if (!user.endTime) {
-
-      // keep growing fontsize
-      let growthRate = 32;
-      let duration = (Date.now() - user.startTime) / 1000;
-      let size = duration * growthRate + 16;
-      textSize(size);
-
-      // turn keycode to actual letter
-      // print the letter
-      text(user.keycode, user.xpos, user.ypos);
-    } else {
-      textSize(32);
-      text(user.keycode, user.xpos, user.ypos);
+    for (let keycode=0; keycode < user.pressed.length; keycode++) {
```

Comment:
IMO an object for `pressed` and an object loop would be better here. If you have time, try the following in terminal:

```
$ node
> let pressed = []
> pressed[65] = { startTime: Date.now() }
> pressed.length     // this should say 66 because it will create empty objects to fill up the array
> for (var i = 0; i < pressed.length; i++) { console.log(i) }
```

vs alternative as an object:

![image](https://user-images.githubusercontent.com/6887902/36109629-f2fb6024-0fed-11e8-9f7a-babe3c398d5d.png)